flink-dist/src/main/flink-bin/kubernetes-bin/kubernetes-session.sh

```diff
@@ -36,4 +36,4 @@ log_setting="-Dlog.file="$log" -Dlog4j.configuration=file:"$FLINK_CONF_DIR"/log4
 
 export FLINK_CONF_DIR
 
-$JAVA_RUN $JVM_ARGS -classpath "$CC_CLASSPATH" $log_setting org.apache.flink.kubernetes.cli.FlinkKubernetesCustomCli "$@"
+$JAVA_RUN $JVM_ARGS -classpath "$CC_CLASSPATH" $log_setting org.apache.flink.kubernetes.cli.KubernetesSessionCli "$@"
```

Comment:
I think we need to add `-e kubernetes-session-cluster` here. So when we use `kubernetes-session.sh` to start a session cluster, we do not need to always add `-e kubernetes-session-cluster`. It is strange for users.

```
$JAVA_RUN $JVM_ARGS -classpath "$CC_CLASSPATH" $log_setting org.apache.flink.kubernetes.cli.KubernetesSessionCli "-e" "kubernetes-session-cluster" "$@"
```

Reply:
You are right, I am fixing this.

Reply:
You are right that we need to specify the target here but I will add it in the `KubernetesSessionCli.getEffectiveConfiguration()` because whenever the user explicitly goes to this class, he/she intends to use this executor.

Reply:
Make sense to me.

Reply:
Good catch! Thanks a lot for checking it. I pushed a commit and added a test. Actually the change removes the test because I think that the assumption behind the `checkState` is pretty restrictive.